### DISCUSSION TITLE
Ugrade 3.7 -> 4.0

### DISCUSSION THREAD

**Discussion by CvekCoding** [mediagis/nominatim-docker#317] [General]
Hello.
I didnt find any info how to upgrade an existing installation of Nominatim 3.7 to 4.0. Can you advise me how I can do this?

Thanks in advance).

**Comment by philipkozeny** [comment]
I have not tested this, but I would

1. start a 4.0 docker container with the volumes from the 3.7 version
2. run something like `docker exec -it nominatim /bin/bash` to access the container
3. run the update as it is described here: https://nominatim.org/release-docs/latest/admin/Migration/#database-migrations
4. hope for the best :)

**Comment by CvekCoding** [comment]
Doesnt work because 4.0 cant even start withouit of icu tokenizer. But 3.7 doesnt have it
